### ISSUE TITLE
Can't find RiotSharp.CurrentGameEndpoint.Enums in RiotSharp.dll

### ISSUE THREAD

**Issue by simonti99** [BenFradet/RiotSharp#674]
Hello, I need to find **Queue Type Enumerator** because it could help me to convert queue_id into its own queue type string.
I try to resolve RiotSharp.CurrentGameEndpoint.Enums [like the documentation suggests](https://benfradet.github.io/RiotSharp/api/RiotSharp.CurrentGameEndpoint.Enums.GameQueueType.html#RiotSharp_CurrentGameEndpoint_Enums_GameQueueType), but I don't find anything.

Can anyone help me? Thanks.

**Comment by AoshiW** [comment]
The documentation is very outdated (last update Nov 2016!).
Here are QueueID http://static.developer.riotgames.com/docs/lol/queues.json

**Comment by simonti99** [comment]
Thank you, I'll use it. By the way is there something we can do to update the documentation? I've read other issues about it but nobody answers...

**Comment by BenFradet** [comment]
yes the whole ci needs to be updated so that it passes, publishes a nuget and publishes the documentation
